### PR TITLE
Product API documentation changes

### LINKED ISSUE #76: change from url to uri and added field url_type + videos
As explained here: https://github.com/CruxConnect/crux-api/issues/751

The api docs need to be updated in sync with the release ^.

### LINKED ISSUE #164: product attribute name changes
```
custom_attributes --> item_attributes
distinguishing_attributes --> sku_distinguishing_attributes
nondistinguishing_attributes --> sku_non_distinguishing_attributes
```

The product attributes above will need to be changed when https://github.com/CruxConnect/crux-api/issues/1261 is released.

### PR DESCRIPTION
**Pull request by nickcoleman**
Updates docs to reflect changes made in Crux-UI for Fizzgig.  
Changes: 
- [x] product url to uri and uri_type syntax. 
- [x] product attributes

fixes #76 
fixes #164

### CODE REVIEW

**Comment by Gidgidonihah** [comment]
Not merging yet as these things go live immediately. We'll release all the changes at the same time.

File: _documentation/product-import-export-format.md [hunk 1]
```diff
@@ -160,15 +160,15 @@ asin
...
 
-#### Distinguishing Attributes
+#### SKU Distinguishing Attributes
 
-distinguishing_attribute.[N].name
+sku_distinguishing_attribute.[N].name
 : (string(max 128)) Distinguishing Attribute Name
 
-distinguishing_attribute.[N].value
+sku_distinguishing_attribute.[N].value
```
Comment (jtprince) [inline _documentation/product-import-export-format.md]: did we add sku_nondistinguishing_attribute to the import/export doc?  It should also be in there now.

File: _documentation/product-import-export-format.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ custom_attribute.[N].type
 : (string) Custom Attribute Type.  Can be one of: 'str', 'int', 'float', 'bool'
 
 product_images_for_item
-: (string) Urls for the product image.  Individual URLs are comma separated urls. Duplicate urls across sku or item will reference the same ProductImage
+: (string) Uris for the product image.  Individual URIs are comma separated uris. Duplicate uris across sku or item will reference the same ProductImage
```
[the remark this answers is not in the document]
Comment (nickcoleman) [reply _documentation/product-import-export-format.md]: Done

File: _documentation/product-import-export-format.md (hunk 1 above)
Comment (nickcoleman) [reply _documentation/product-import-export-format.md]: done

**Pull request merged**